python/cudf/cudf/tests/test_index.py

```diff
@@ -2058,3 +2058,266 @@ def test_index_set_names_error(idx, level, names):
         lfunc_args_and_kwargs=([], {"names": names, "level": level}),
         rfunc_args_and_kwargs=([], {"names": names, "level": level}),
     )
+
+
+@pytest.mark.parametrize(
+    "idx",
+    [pd.Index([1, 3, 6]), pd.Index([6, 1, 3])],  # monotonic  # non-monotonic
+)
+@pytest.mark.parametrize("key", list(range(0, 8)))
+@pytest.mark.parametrize(
+    "method", [None, "ffill", "pad", "bfill", "backfill", "nearest"]
```

Comment:
Could we get away with just testing `ffill` and `bfill` for these tests instead of both method aliases?